selendroid-test-app/src/main/java/io/selendroid/testapp/WebViewActivity.java

```diff
@@ -42,6 +46,27 @@ public void onCreate(Bundle savedInstanceState) {
     setContentView(io.selendroid.testapp.R.layout.webview);
 
     mainWebView = (WebView) findViewById(io.selendroid.testapp.R.id.mainWebView);
+    WebSettings settings = mainWebView.getSettings();
+    settings.setJavaScriptEnabled(true);
+    settings.setDomStorageEnabled(true); // LocalStorage, SessionStorage
+
+    // The following is required to enable WebSQL on pre-Kitkat, per
+    // http://pouchdb.com/errors.html#android_pre_kitkat
+    settings.setDatabaseEnabled(true);
+    // On Kitkat and above, the following will be ignored, and it will write 
+    // to the "database" directory anyway.
+    String databasePath = getDir("database", Context.MODE_PRIVATE).getPath();
+    settings.setDatabasePath(databasePath);
+    // On Kitkat and above, the following will be ignored.
+    webView.setWebChromeClient(new WebChromeClient() {
```

Comment:
Yeah, I got an error trying to do that: `Could not resolve dependencies for project io.selendroid:selendroid-server:apk:0.16.0-SNAPSHOT: Failure to find org.apache.cordova:cordova:jar:android:3.7.0`. So just kept hoping I could code blind and make it work... :stuck_out_tongue: